bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -720,13 +732,14 @@ public Image(Device device, ImageDataProvider imageDataProvider) {
 }
 
 /**
- * Refresh the Image based on the zoom level, if required.
+ * Update zoom and refresh the Image based on the zoom level, if required.
+ *
+ * @param deviceZoomLevel zoom level the image shall be scaled for
  *
  * @return true if image is refreshed
  */
-boolean refreshImageForZoom () {
+boolean handleDPIChange (int deviceZoomLevel) {
```

Comment:
nit: this seems to be the only method using the term "zoomLevel" instead of "zoom"